runtime/libia2/include/permissive_mode.h

```diff
@@ -214,7 +214,7 @@ void permissive_mode_handler(int sig, siginfo_t *info, void *ctxt) {
 static bool exiting IA2_SHARED_DATA = false;
 
 // Process-specific name for log file
-static char log_name[256] IA2_SHARED_DATA = {0};
```

Comment:
I don't think this change is necessary, right?

Reply:
I'm not sure.  Do you remember why you made it?